shared.utils.ts

```diff
@@ -0,0 +1,45 @@
+import { readFileSync } from 'node:fs';
+
+/**
+ * Read a JSON file with canister IDs and transform it into a dictionary. Optionally add prefix to all dictionary keys.
+ */
+export const readCanisterIdsFromJSONFile = ({
+	filePath,
+	prefix
+}: {
+	filePath: string;
+	prefix?: string;
+}): Record<string, string> => {
+	try {
```

Comment:
Nothing changed in the logic of this function correct? Simply refactoring and it does need a review?

Reply:
Yep, I only extracted this function into a separate file.